backend/databases/sql.py

```
@@ -0,0 +1,216 @@
+                "quantity": row[5],
+                "location": row[6],
+                "storage_box": row[7],
+                "brand": row[8],
+                "size": row[9],
+                "condition": row[10],
+                "purchase_date": row[11],
+                "image_data": row[12],
+                "metadata": row[13],
+                "created_at": row[14],
+                "last_updated": row[15]
+            })
```

Ruff (0.16.0)

[warning] Use a list comprehension to create a transformed list

(PERF401)